deps_licenses/licenses_mac_intel_user.txt

```diff
@@ -31,20 +31,20 @@ protobuf, 3.20.3, BSD-3-Clause
 psutil, 5.9.8, BSD License
 python-dateutil, 2.9.0.post0, Apache Software License; BSD License
 pytz, 2024.1, MIT License
-requests, 2.31.0, Apache Software License
-scikit-learn, 1.3.2, BSD License
+requests, 2.32.2, Apache Software License
+scikit-learn, 1.1.3, BSD License
 scipy, 1.10.1, BSD License
 six, 1.16.0, MIT License
-skl2onnx, 1.16.0, Apache Software License
+skl2onnx, 1.12, Apache Software License
 skops, 0.5.0, MIT
 skorch, 0.11.0, new BSD 3-Clause
 sympy, 1.12, BSD License
 tabulate, 0.8.10, MIT License
 threadpoolctl, 3.5.0, BSD License
 torch, 1.13.1, BSD License
-tqdm, 4.66.2, MIT License; Mozilla Public License 2.0 (MPL 2.0)
+tqdm, 4.66.4, MIT License; Mozilla Public License 2.0 (MPL 2.0)
 typing_extensions, 4.5.0, Python Software Foundation License
 tzdata, 2024.1, Apache Software License
 urllib3, 2.2.1, MIT License
-xgboost, 1.7.6, Apache Software License
+xgboost, 1.6.2, Apache Software License
```

Comment:
This is weird. It happens for many different libraries. Why would 3.11 not support a later version of xgboost?

Reply:
indeed!